### ISSUE TITLE
Uncaught TypeError: Cannot read property 'get' of undefined

### ISSUE THREAD

**Issue by ThrudUK** [pojome/elementor#3104]
Not sure if this is linked to the latest version of not, however, on one page of my site I am getting a "grey screen" when trying to edit a page.  This is a duplicated page for Polylang but I have no problems with other pages.

You can see a screenshot of the console error here :  https://imgur.com/a/uLpvL

System info is below :

`					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.28 (Unix) OpenSSL/1.0.2g
	MySQL version: 5.7.20
	PHP Version: 7.1.12
	PHP Max Input Vars: 5000
	PHP Max Post Size: 1024M
	GD Installed: Yes
	Elementor Library: Connected

== WordPress Environment ==
	Version: 4.9.1
	Site URL: https://guardians.christchurchwebsolutions.co.uk
	Home URL: https://guardians.christchurchwebsolutions.co.uk
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: Europe/London
	Debug Mode: Inactive

== Theme ==
	Name: Astra Child
	Version: 1.0.0
	Author: Brainstorm Force
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 1.1.2
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.84 Safari/537.36

== Active Plugins ==
	Duplicate Menu
		Version: 0.2.1
		Author: Jonathan Christopher

	Elementor
		Version: 1.8.12
		Author: Elementor.com

	Elementor Pro
		Version: 1.12.2
		Author: Elementor.com

	Header Footer Elementor
		Version: 1.0.8
		Author: Brainstorm Force, Nikhil Chavan

	jQuery in Posts, Pages, Custom Post Types
		Version: 4.0
		Author: WebCodingPlace

	Kraken Image Optimizer
		Version: 2.6.2
		Author: Karim Salman

	Polylang
		Version: 2.2.7
		Author: Frédéric Demarle

	Really Simple SSL
		Version: 2.5.24
		Author: Rogier Lankhorst

	UpdraftPlus - Backup/Restore
		Version: 1.14.2
		Author: UpdraftPlus.Com, DavidAnderson

	Yoast SEO
		Version: 6.0
		Author: Team Yoast

== Debug ==
	Errors: 
		TypeError: Cannot read property 'get' of undefined
			at https://guardians.christchurchwebsolutions.co.uk/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=1.12.2 - 2:27208
			2018-01-07 11:38 +00:00
			x 19 times

		TypeError: Cannot read property 'model' of undefined
			at https://guardians.christchurchwebsolutions.co.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=1.8.11 - 2:142585
			2018-01-05 10:45 +00:00
			x 5 times

		ReferenceError: jquery is not defined
			at https://guardians.christchurchwebsolutions.co.uk/?elementor-preview&1513868949718 - 485:7
			2017-12-21 15:09 +00:00

		TypeError: Cannot read property 'model' of undefined
			at https://guardians.christchurchwebsolutions.co.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=1.8.9 - 2:142459
			2017-12-18 15:27 +00:00

		TypeError: Cannot read property 'children' of undefined
			at https://guardians.christchurchwebsolutions.co.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=1.8.9 - 2:141524
			2017-12-15 17:42 +00:00
			x 4 times

		TypeError: Cannot read property 'errors' of undefined
			at https://guardians.christchurchwebsolutions.co.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=1.8.9 - 2:114377
			2017-12-11 16:37 +00:00
			x 3 times

				`

**Comment by ThrudUK** [comment]
A little update to this.

I used duplicate post to create a copy of the page and then edit and it works fine. 

HOWEVER

I then changed the language manually via the edit page which puts the page in the language sub fodler and Elementor wont load.

**Comment by ThrudUK** [comment]
So, the work round  is : 
duplicate the page
rename it
edit it
save it
then set the language

Interesting to note this only seems to be a problem with pages that have pro elements on them

**Comment by bainternet** [comment]
@ThrudUK 

Please contact our support and see how they can help. support@elementor.com

**Comment by nicomollet** [comment]
@ThrudUK I just had the same problem, with Polylang.

My Elementor page had a global widget. This Elementor Library widget didn't have a (Poylang) language set.

I had to go to Polylang Languages settings page `/wp-admin/admin.php?page=mlang` and "set default language for contents without language" or something like that.

Now the Elementor Editor is loading correctly when the page contains global widgets.

**Comment by galaxygarden** [comment]
Just had similar issue using WPML. Was about to surrender and came across this thread, which pointed me in the right direction. When editing a translated page with Elementor, the page builder simply wouldn't load (or would occasionally load in safe mode). I'm using a couple of global widgets and they were not translated, so I went to Templates > Saved Templates and translated the global widgets into the other languages. Problem solved.

Side note for those that may be googling for the same thing that I was: this issue only popped up after I changed the default language in WPML - no idea why.
